src/snarks/groth16/mpc/phase2.cpp

```diff
@@ -0,0 +1,66 @@
+#include "snarks/groth16/mpc/phase2.hpp"
+
+namespace libzeth
+{
+
+template<>
+void srs_mpc_phase2_accumulator<libff::alt_bn128_pp>::write_compressed(
+    std::ostream &out) const
+{
+    using G1 = libff::alt_bn128_G1;
+    check_well_formed(*this, "mpc_layer2 (write)");
+
+    // Write fixed-sizes cs_hash and sizes first.
```

Comment:
`Write fixed-sizes cs_hash` -> Not sure I understand. What do you mean by "fixed-sizes cs_hash"?

Reply:
Sorry, that was a typo ... should have been `fixed-size cs_hash`.  I've reworded the comment.